lara-typescript/src/section-authoring/api/util/move-utils.tsx

```diff
@@ -0,0 +1,91 @@
+import { set } from "immer/dist/internal";
```

Comment:
I didn't see this import being used below (and it's a bit odd to be importing from an internal file).

Reply:
Oof sometimes VSCode adds imports, and I don't always notice.  Thanks for catching it.